obj.js

```diff
@@ -5,10 +5,36 @@ function getClassName(obj) {
   return (proto && (proto[Symbol.toStringTag] || proto.constructor.name)) || "Object";
 }
 
+function isPrivateProperty(propName) {
+  return propName.startsWith("_") || propName == "constructor";
+}
+
+// This is only set as a function on classes that are known to be observers
+function subscribe(subscriber) {
+  subscriber(this);
+  if (!this._subscribers) {
+    this._subscribers = new Set();
+  }
+  this._subscribers.add(subscriber);
+  return () => this._subscribers.remove(subscriber);
+}
+
 class RemoteClass {
   constructor(className) {
     this.className = className;
   }
+  _notifySubscribers() {
```

Comment:
Can we please express this as subclass `RemoteObservableClass`, which then has functions `_notifySubscribers()` and `subscribe()`?

Reply:
> Can we please express this as subclass `RemoteObservableClass`, which then has functions `_notifySubscribers()` and `subscribe()`?

(This is going to be all the more important once we add more complex objects like the collections.)

Reply:
This won't work for an observable class that's a subclass of a regular class.